sdk/Lusid.Sdk.Tests/tutorials/Ibor/Portfolios.cs

```diff
@@ -253,6 +254,7 @@ public void Add_Transactions_To_Portfolio_With_Property()
         
         [LusidFeature("F2-4")]
         [Test]
+        // TODO: These tests names need to be renamed and be in line with the standard https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices
```

Comment:
these are intended to be working examples rather than tests per se, I dont think they need to follow test naming convention

Reply:
Yes I agree now, I'll remove the comment